commit 97eda62bcc17cdb499f32db50bba2f267e2d922a
Author: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
Date:   Thu Jan 10 19:54:40 2019 +0100

    overlays: update RPi's firmware based touchscreen
    
    The driver was included in 5.0-rc1, we adapt the ovrelay to the upstream
    bindings.
    
    Signed-off-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>

diff --git a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
index 1915ce188bf3..baa8cda1a9e1 100644
--- a/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-ft5406-overlay.dts
@@ -7,24 +7,21 @@
 	fragment@0 {
 		target-path = "/";
 		__overlay__ {
-			rpi_ft5406: rpi_ft5406 {
-				compatible = "rpi,rpi-ft5406";
-				firmware = <&firmware>;
-				status = "okay";
-				touchscreen-size-x = <800>;
-				touchscreen-size-y = <480>;
-				touchscreen-inverted-x = <0>;
-				touchscreen-inverted-y = <0>;
-				touchscreen-swapped-x-y = <0>;
+			firmware {
+				ts: touchscreen {
+					compatible = "raspberrypi,firmware-ts";
+					touchscreen-size-x = <800>;
+					touchscreen-size-y = <480>;
+				};
 			};
 		};
 	};
 
 	__overrides__ {
-		touchscreen-size-x = <&rpi_ft5406>,"touchscreen-size-x:0";
-		touchscreen-size-y = <&rpi_ft5406>,"touchscreen-size-y:0";
-		touchscreen-inverted-x = <&rpi_ft5406>,"touchscreen-inverted-x:0";
-		touchscreen-inverted-y = <&rpi_ft5406>,"touchscreen-inverted-y:0";
-		touchscreen-swapped-x-y = <&rpi_ft5406>,"touchscreen-swapped-x-y:0";
+		touchscreen-size-x = <&ts>,"touchscreen-size-x:0";
+		touchscreen-size-y = <&ts>,"touchscreen-size-y:0";
+		touchscreen-inverted-x = <&ts>,"touchscreen-inverted-x:?";
+		touchscreen-inverted-y = <&ts>,"touchscreen-inverted-y:?";
+		touchscreen-swapped-x-y = <&ts>,"touchscreen-swapped-x-y:?";
         };
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1153_97eda62bcc17cdb499f32db50bba2f267e2d922a.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1153_97eda62bcc17cdb499f32db50bba2f267e2d922a.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
